docs/core/deploying/single-file/overview.md

```diff
@@ -158,13 +154,33 @@ Single file applications have all related PDB files alongside the application, n
 
 Managed C++ components aren't well suited for single file deployment. We recommend that you write applications in C# or another non-managed C++ language to be single file compatible.
 
-### Output differences from .NET 3.x
+### Native libraries
+
+Only managed DLLs are bundled with the app into a single executable. When the app starts, the managed DLLs are extracted and loaded in memory, avoiding the extraction to a folder. This approach means that the managed binaries are embedded in the single file bundle, but the native binaries of the core runtime itself are separate files.
 
-In .NET Core 3.x, publishing as a single file produced one file, consisting of the app itself, dependencies, and any other files in the folder during publish. When the app starts, the single file app was extracted to a folder and run from there.
+To embed those files for extraction and get one output file, set the property `IncludeNativeLibrariesForSelfExtract` to `true`. For more information about extraction, see [Including native libraries](#include-native-libraries).
 
-Starting with .NET 5, only managed DLLs are bundled with the app into a single executable. When the app starts, the managed DLLs are extracted and loaded in memory, avoiding the extraction to a folder. On Windows, this approach means that the managed binaries are embedded in the single file bundle, but the native binaries of the core runtime itself are separate files.
+Specifying `IncludeAllContentForSelfExtract` extracts all files, including the managed assemblies, before running the executable. This may be helpful for rare application compatibility problems.
 
-To embed those files for extraction and get one output file, like in .NET Core 3.x, set the property `IncludeNativeLibrariesForSelfExtract` to `true`. For more information about extraction, see [Including native libraries](#include-native-libraries).
+ [!NOTE]
```

Comment:
```suggestion
 [!IMPORTANT]
```

When we have two alerts back-to-back, it's best to vary their styling to help differentiate between them, if they cannot be otherwise avoided.